concepticondata/concepticon.tsv

```diff
@@ -1150,12 +1150,12 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1149	ORPHAN	Kinship	A person one or both of whose parents have died, especially a minor.	Person/Thing	
 1150	THUNDER	The physical world	Sound produced by fast air expansion induced by a lightning.	Person/Thing	
 1151	VEIL	Clothing and grooming	A head covering.	Person/Thing	
-1152	BEGET	The body	(especially of a male parent) to procreate or generate.	Action/Process	
+1152	BEGET	The body	(Especially of a male parent) to procreate or generate.	Action/Process	
```

Comment:
I would capitalize the "to" after the parentheses as well.

Reply:
All the other cases with this sequence are lowercase: "(....) to".